esp/templates/program/modules/admincore/directory.html

```diff
@@ -152,31 +152,22 @@ <h1>Admin Portal for {{ program.niceName }} (ID: {{ program.id }})</h1>
 </tr>
 </table>
 
-<div class="module_group_header">Complete Module List</div>
-<div id="complete_list">
+<div class="module_group_header">Additional Modules</div>
+<div id="additional_modules">
 <table border="0" width="100%">
 <tr>
  <td width="40%">
 <ul>
-<li><a href="/onsite/{{program.getUrlBase}}/main">
-     Go To On-Site Page
-    </a>
-</li>
-
 {% for module in modules %}
 {% if module.isStep %}
-  {% if not module.useTemplate %}
-    <li>{% autoescape off %}{{ module.makeLink }}{% endautoescape %}</li>
-  {% else %}
-    <li><span onmouseover="updateDocs('{{module.docs|linebreaks|addslashes}}');"><em>{{ module.module.link_title }} (part of Dashboard)</em></span></li>
-  {% endif %}
+    {% if not module.isAdminPortalFeatured %}
+      {% if not module.useTemplate %}
+        <li>{% autoescape off %}{{ module.makeLink }}{% endautoescape %}</li>
+      {% endif %}
+    {% endif %}
 {% endif %}
 {% endfor %}
-<li><a href="/manage/{{ one }}/{{ two }}/deadline_management/" title="Deadline Management" onmouseover="updateDocs('<p>Edit Program Deadlines</p>');">Edit Program Deadlines</a></li>
-<li><a href="/manage/{{ one }}/{{ two }}/lunch_constraints/" title="Setup Lunch Constraints" onmouseover="updateDocs('<p>Setup Lunch Constraints</p>');">Setup Lunch Constraints</a></li>
-<li><a href="/manage/{{ one }}/{{ two }}/registrationtype_management/" title="Registration Type Management" onmouseover="updateDocs('<p>Registration Type Management</p>');">Registration Type Management</a></li>
-<li><a href="/learn/{{ one }}/{{ two }}/studentreg/" title="Student Registration" onmouseover="updateDocs('<p>Student Reg</p>');">Student Reg</a></li>
-<li><a href="/learn/{{ one }}/{{ two }}/catalog/" title="Catalog" onmouseover="updateDocs('<p>Catalog</p>');">Catalog</a></li>
+<li><a href="/manage/{{ program.getUrlBase }}/surveys">View Surveys</a></li>
```

Comment:
If the survey modules are always included this will probably be fine, but if not, we'll want to check the modules are enabled before including this line. (Is there not a way to just make it work with the previous forloop?)

Reply:
Same thing with the Teacher Training and Interviews link at the top of this page.

Reply:
good point. The teacher training is always included, but the surveys are not. This ties into the bigger question of how we want to reorganize this page. Is there any consequence to including the surveys and them going unused? Will they get in the way at any point?

Reply:
Well, I guess we'll always be including surveys now (see above comment), so this is probably a null issue (although I still think it would be cleaner if they are included in the for loop above this)

Reply:
Yeah, but the URL isn't the default manage/.../teachersurvey, and all survey modules all point to the same link, so I'm tempted to punt that to the redo of this page

Reply:
I'm not sure I follow, here you're using program.getUrlBase, but you could totally do /manage/one/two/surveys.

Reply:
OK, well if we're gonna punt it, let's at least move the survey link up into one of the featured areas at the top of this page, that way this part is only the forloop. That will also help with trying to promote the new survey changes.

Reply:
where do you have in mind? Under "Quicklinks" good?

Reply:
commit a1c83e7